x-pack/plugins/maps/public/lens/choropleth_chart/setup.ts

```diff
@@ -34,7 +34,10 @@ export function setupLensChoroplethChart(
     try {
       emsFileLayers = await getEmsFileLayers();
     } catch (error) {
-      // ignore error, lack of EMS file layers will be surfaced in dimension editor
+      // eslint-disable-next-line no-console
+      console.warn(
+        `Lens region map is unable to access boundaries from Elastic Maps Service (EMS). Set kibana.yml 'map.includeElasticMapsService: false' to avoid unnecessary EMS requests. For more details please visit ${coreStart.docLinks.links.maps.connectToEms}`
```

Comment:
nit

```suggestion
        `Lens region map is unable to access boundaries from Elastic Maps Service (EMS). To avoid unnecessary EMS requests, set 'map.includeElasticMapsService: false' in 'kibana.yml'. For more details please visit ${coreStart.docLinks.links.maps.connectToEms}`
```